cdc/sorter/unified/heap.go

```diff
@@ -25,10 +25,19 @@ type sortHeap []*sortItem
 func (h sortHeap) Len() int { return len(h) }
 func (h sortHeap) Less(i, j int) bool {
 	if h[i].entry.CRTs == h[j].entry.CRTs {
-		if h[j].entry.RawKV.OpType == model.OpTypeResolved && h[i].entry.RawKV.OpType != model.OpTypeResolved {
+		if h[j].entry.RawKV.OpType == model.OpTypeResolved {
```

Comment:
Consider two `Resolved` events A and B with the same `CommitTs`, before this change,  less(A, B) would be `false`, but after this changed, it would be `true`.